src/main/java/wtf/g4s8/examples/system/Decision.java

```diff
@@ -0,0 +1,8 @@
+package wtf.g4s8.examples.system;
+
+public enum Decision {
+    NONE,
```

Comment:
@stepanvalyavskiy what is a difference between none and unkown?

Reply:
No diff, unknown is unused and removed.